test/tests/fs/xhrFS/listing.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Unit tests for MountableFileSystem's mount/unmount features.
+ */
+var fs = require('fs'),
+    assert = require('wrapped-assert');
+
+module.exports = function() {
+  var oldRootFS = fs.getRootFS();
+
+  var rootForXFS = new BrowserFS.FileSystem.InMemory();
```

Comment:
I was following the mountable-fs template, without really understanding what that test was about. Will fix it.